src/main/webapp/dev/js/common/instructor.js

```diff
@@ -310,13 +310,14 @@ function bindDeleteButtons() {
         const courseId = $button.data('courseid');
         const feedbackSessionName = $button.data('fsname');
 
-        const messageText = `Are you sure you want to delete the feedback session ${feedbackSessionName} in ${courseId}?`;
+        const messageText = `Are you sure you want to delete the feedback session ${feedbackSessionName} in ${courseId}? `
+                + 'This action can be reverted by going to the "sessions" tab and restoring the desired session(s).';
```

Comment:
Nit-picking as usual: the "*Sessions*" tab

Reply:
Will also change the "courses" below :)